app/payloads/api/v1/schema/models.rb

```diff
@@ -184,6 +184,35 @@ def self.login_user
       required: %w[phone_number password otp] }
   end
 
+  def self.appointment
+    { type: :object,
+      properties: {
+        id: { '$ref' => '#/definitions/uuid' },
+        patient_id: { '$ref' => '#/definitions/uuid' },
+        facility_id: { '$ref' => '#/definitions/uuid' },
+        date: { type: :string, format: :date },
+        status: { type: :string, enum: Appointment.statuses.keys },
+        status_reason: { type: :string, enum: Appointment.status_reasons.keys },
+        created_at: { '$ref' => '#/definitions/timestamp' },
+        updated_at: { '$ref' => '#/definitions/timestamp' } },
+      required: %w[id patient_id facility_id date status status_reason created_at updated_at]
+    }
+  end
+
+  def self.communication
+    { type: :object,
+      properties: {
+        id: { '$ref' => '#/definitions/uuid' },
+        appointment_id: { '$ref' => '#/definitions/uuid' },
+        user_id: { '$ref' => '#/definitions/uuid' },
+        communication_type: { type: :string, enum: Communication.communication_types.keys },
+        communication_result: { type: :string, enum: Communication.communication_results.keys },
```

Comment:
We could transform `type` and `result` with prefixes, so the API can be elegant.